### ISSUE TITLE
Remembering location of an item on the map

### ISSUE THREAD

**Issue by ggttttggggtt** [CleverRaven/Cataclysm-DDA#59917]
### Is your feature request related to a problem? Please describe.

I often drop my backpack to fight with less encumbrance, and then struggle to find it later because I forgot where I dropped it.

### Solution you would like.

Maybe something like "L - Remember location of item on map" on this menu, similar to the option for vehicles?
![image](https://user-images.githubusercontent.com/87790164/182998499-b361e4b5-20a4-4fde-99b6-d5928f612094.png)
When this is selected for an item, an autonote would automatically be created at its location when it gets dropped, and removed when picked up.

### Describe alternatives you have considered.

Simply having better memory

**Comment by descan** [comment]
If this functionality were tied to [f]avouriting items, that might be useful. If I drop an item I've favourited, it usually was an accident, or I'm going to want to come back for it. Making sure I've set them all to remember location would be tedious - especially if I don't know that I've dropped or will drop it, like a gun I never want to not have on (and therefore might not think to mark its location if dropped).
